frontend/src/v5/ui/components/dashboard/dashboardList/dashboardList.styles.ts

```diff
@@ -50,25 +50,25 @@ export const Divider = styled(DividerComponent)`
 `;
 
 export const DashboardListButton = styled(Button).attrs({
-	variant: 'outlined',
-	color: 'primary',
+	variant: 'contained',
+	color: 'secondary',
 })`
 	${({ theme }) => theme.typography.h3}
+	/* TODO - fix when new palette is released */
+	background-color: #E8EEF8;
```

Comment:
I'snt the background colour just the primary colour with a different opacity? (consult with @george-hadfield) 
in that case we can have a helper funtion for changing the opacity to a colour.

Reply:
The issue written up by George says "_Background-color: New hex (#E8EEF8)_"

Reply:
I think you should re consult with him about it. I think he is just colour picking the colour and not thinking about what it actually means.

Reply:
And if the case is that he actually wants a solid colour we should add it to the palette now, so you would need to consult with George again about what it actually means and if he has the concept of that colour in a different palette